transactions/transactor.go

```diff
@@ -24,8 +25,13 @@ const (
 	sendTxTimeout = 300 * time.Second
 
 	defaultGas = 90000
+
+	validSignatureSize = 65
```

Comment:
Can we take this value from go-ethereum package?

Reply:
@adambabik I see it's actually hardcoded there: 

https://github.com/status-im/status-go/blob/develop/vendor/github.com/ethereum/go-ethereum/core/types/transaction_signing.go#L196